zuul-core/src/main/java/com/netflix/netty/common/accesslog/AccessLogChannelHandler.java

```diff
@@ -100,6 +109,7 @@ public void userEventTriggered(ChannelHandlerContext ctx, Object evt) throws Exc
                         state.dateTime,
                         localPort,
                         remoteIp,
```

Comment:
Decided to keep this one as it can provide extra visibility. For example when requests travel through CLBs